packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -98,6 +98,7 @@ export class DashboardModel {
 
     private static async createVersion(
         trx: Transaction,
+        dashboardUuid: string,
```

Comment:
is this not optional ? `dashboardUuid: string | undefined`

Reply:
Even though it might not be used, we don't know at this point, so it is mandatory. Note this is a method to create a dashboard version, not a chart version.